src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -551,6 +591,121 @@ public UpdateItemResult updateItem(UpdateItemRequest updateItemRequest) {
+                    .getTenantTableBackupFromArn(restoreTableFromBackupRequest.getBackupArn());
+            RestoreMtBackupRequest mtRestoreRequest = new RestoreMtBackupRequest(backupMetadata.getBackupName(),
+                new TenantTable(backupMetadata.getVirtualTableName(), backupMetadata.getTenantId()),
+                new TenantTable(restoreTableFromBackupRequest.getTargetTableName(), getMtContext().getContext()));
+            TenantRestoreMetadata restoreMetadata = backupManager.get().restoreTenantTableBackup(
+                mtRestoreRequest,
+                getMtContext());
+
+            return new RestoreTableFromBackupResult().withTableDescription(
+                mtTableDescriptionRepo.getTableDescription(restoreMetadata.getVirtualTableName()));
+        } else {
+            throw new ContinuousBackupsUnavailableException("Backups can only be created by configuring a backup "
+                + "managed on an mt-dynamo table builder, see <insert link to backup guide>");
+        }
+    }
+
+    @Override
+    public DescribeBackupResult describeBackup(DescribeBackupRequest describeBackupRequest) {
+        if (backupManager.isPresent()) {
+            if (getMtContext().getContextOpt().isPresent()) {
+                throw new UnsupportedOperationException("TODO: Implement tenant table backup describe");
+            }
+            MtBackupMetadata backupMetadata = getBackupManager().getBackup(describeBackupRequest.getBackupArn());
+            if (backupMetadata != null) {
+                return MtBackupAwsAdaptorKt.getBackupAdaptorSingleton().getDescribeBackupResult(backupMetadata);
+            } else {
+                throw new BackupNotFoundException("No backup with arn "
+                    + describeBackupRequest.getBackupArn() + "found");
+            }
+        } else {
+            throw new ContinuousBackupsUnavailableException("Backups can only be created by configuring a backup "
+                + "managed on an mt-dynamo table builder, see <insert link to backup guide>");
+        }
+    }
+
+    @Override
+    public CreateBackupResult createBackup(CreateBackupRequest createBackupRequest) {
+        if (backupManager.isPresent()) {
+            Preconditions.checkNotNull(createBackupRequest.getBackupName(), "Must pass backup name.");
+            Preconditions.checkArgument(createBackupRequest.getTableName() == null,
+                "Multitenant backups cannot backup individual tables, table name arguments are disallowed.");
+            backupManager.get().createBackup(createBackupRequest);
+
+            ExecutorService executorService = Executors.newFixedThreadPool(mtTables.keySet().size());
+
+            Set<String> snapshottedTables = Sets.newHashSet();
+            List<Future<SnapshotResult>> futures = Lists.newArrayList();
+            Set<String> origMtTables = ImmutableSet.copyOf(mtTables.keySet());
```

Comment:
What about the metadata table? Do we need to take a snapshot of that one as well?

Reply:
Given we don't allow updates to existing virtual table metadata, I don't think we do need a snapshot. A row either appears, or does not, and for both cases, missing metadata for virtual data should be handled as an orphaned backup, and vice versa as an empty backup